ocaml/xapi/message_forwarding.ml

```diff
@@ -1679,8 +1679,12 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
     let assert_can_migrate_sender ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~vgpu_map ~options =
       info "VM.assert_can_migrate_sender: VM = '%s'" (vm_uuid ~__context vm);
       let local_fn = Local.VM.assert_can_migrate_sender ~vm ~dest ~live ~vdi_map ~vif_map ~vgpu_map ~options in
-      forward_vm_op ~local_fn ~__context ~vm
-        (fun session_id rpc -> Client.VM.assert_can_migrate_sender rpc session_id vm dest live vdi_map vif_map vgpu_map options)
+      try
+        forward_vm_op ~local_fn ~__context ~vm
+          (fun session_id rpc -> Client.VM.assert_can_migrate_sender rpc session_id vm dest live vdi_map vif_map vgpu_map options)
+      with
+      | Api_errors.Server_error(code, params) when code=Api_errors.message_method_unknown ->
+        debug "VM.assert_can_migrate_sender not known by destine, assuming older version host no need to do this."
```

Comment:
I think the log level should at least be `warning` and it's better to replace the `params` with `_` in the pattern match statement.

Reply:
No, the current "cannot marshall arguments for..." error is only debug, so warning is not necessary. Just keep the two similar handling the same level.

Reply:
Just followed the same style for the (code, params) from somewhere else, not used params in this style is also seen from various codes, so not infringing either.

Reply:
1. The log level of this kind of log is something we don't expect, so it's better to log all of them with `warning` level.
2. I prefer using `_` instead named variable for useless variable because it makes the code clean and easy to read (I need to remember less variables when reading the code...)

Reply:
Grammar: "VM.assert_can_migrate_sender not known by **destination, assuming it can ignore this check**". I agree that this should be at priority warning.